src/views/Login.vue

```diff
@@ -102,12 +114,19 @@ export default class Login extends Vue {
   valid = true
   error = false
   loading = false
+  source = 'moonraker'
+  availableSources = [this.source]
+
+  async mounted () {
+    const authInfo = await this.$store.dispatch('auth/getAuthInfo')
+    Object.assign(this, authInfo)
```

Comment:
Personal opinion here, but I was never a big fan of this type of approach as it hides what is actually being updated on the target. Nevertheless, this is correct, so all good!